### PR TITLE
Created French translation

### PR DESCRIPTION
**Pull request by rene-coty**
I created the fr.po file + updated the LINGUAS & the Credits files

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+msgid "Light"
+msgstr "Clair"
+
+#: src/preferences_window.py:280
+msgid "Dark"
+msgstr "Sombre"
+
+#: src/preferences_window.py:291
+msgid "Profile:"
+msgstr "Profil :"
```
Comment (ghost) [inline po/fr.po]: Could you change this to "Profil:"

**Comment by ghost** [comment]
Was I right to take away the spaces after the "Profil"?

**Comment by ghost** [comment]
The translation seems fine otherwise.

**Comment by rene-coty** [comment]
Hello! In French we always put a non-breaking space between a word and some punctuation marks like ";", ":" or "!/?". It is better to leave the space..

**Comment by ghost** [comment]
> Hello! In French we always put a non-breaking space between a word and some punctuation marks like ";", ":" or "!/?". It is better to leave the space..

Sorry! I'll go ahead and reverse it, then

**Comment by ghost** [comment]
Alright, I fixed the spaces now, so this PR seems ready to merge. @rene-coty Thanks for the contribution!

**Pull request merged**